src/breakpoints/model.ts

```diff
@@ -11,6 +11,18 @@ import { IDebugger } from '../tokens';
  * A model for a list of breakpoints.
  */
 export class BreakpointsModel implements IDisposable {
+  /**
+   * Get the last breakpoints from story
```

Comment:
Does "story" here refer to the history of breakpoints?

Reply:
Exactlly, `id` which is set here, i use to get current breakpoints cell's from story.